internal/util/diff/pkgdiff.go

```diff
@@ -120,3 +132,28 @@ func pkgSet(pkgPath string) (sets.String, error) {
 	}
 	return pkgFiles, nil
 }
+
+// addMergeCommentToExpected copies the expected path directory contents to
+// new temp directory and adds merge comment to the resources in directory
+// it also returns the cleanup function to clean the created temp directory
+func addMergeCommentToExpected(path string) (string, func(), error) {
+	expected, err := ioutil.TempDir("", "")
+	if err != nil {
+		return "", nil, err
+	}
+	err = copyutil.CopyDir(path, expected)
```

Comment:
Do we absolutely need to do a copy here? I have noticed that file copying actually matters for how long it takes for kpt to get/update a package, so we should try to minimize if possible.

Reply:
Good catch. We don't need it. Removed it.